src/pres/loader/Filters.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react'
 import {getNumberIcon} from './Common'
-import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
```

Comment:
There was an error in the console about this component being removed, so I followed their recommendation by changing it to `AccordionDetails`.

Reply:
It looked like a simple fix but sorry if I missed an obvious reason why this hadn't already been fixed.

Reply:
When I tried this last I ran into some issues, Compile errors iirc. If it is compiling, i think we are good.